### ISSUE TITLE
Rozmyty interfejs (chyba powiązane z #5)

### ISSUE THREAD

**Issue by kanlukasz** [lkosson/profak#7]
Cześć,
Dzięki za szybką aktualizację.

Testuje nową wersje, ale coś jest nie tak, bo teraz wszystko jest całkiem rozmyte.
Poniżej porównanie 1.8 do 1.7

![image](https://github.com/user-attachments/assets/e9e105c0-d361-47bc-ae04-84a6c2b62ddd)

( chyba powiązane z #5 )

**Comment by lkosson** [comment]
Tak, to ma związek z #5. Tamto rozwiązanie jest przejściowe póki nie poprzerabiam wszystkich ekranów tak, by się natywnie skalowały wraz z DPI.

**Comment by lkosson** [comment]
Teraz powinno być lepiej. Sprawdzisz na aktualnej wersji z repo? Ewentualnie podmień pliki z dołączonej paczki: [profak-1.8.0-fix-20241004.zip](https://github.com/user-attachments/files/17256713/profak-1.8.0-fix-20241004.zip)

**Comment by kanlukasz** [comment]
Podmieniłem pliki na z fixa i teraz wygląda idealnie. Super!

**Comment by lkosson** [comment]
Poprawione w wersji 1.8.1.
